sql/core/src/test/scala/org/apache/spark/sql/FileBasedDataSourceSuite.scala

```diff
@@ -842,6 +842,142 @@ class FileBasedDataSourceSuite extends QueryTest
       }
     }
   }
+
+  test("SPARK-31116: Select simple parquet with case insensitive mode") {
```

Comment:
Hi, @kimtkyeom . Please remove this irrelevant test case. This does not fail.

Reply:
Ah, okay. I removed it.